rose/client/car.py

```diff
@@ -9,7 +6,6 @@ def __init__(self, id, x, y):
         self.id = id
         self.x = None
         self.y = None
```

Comment:
One off-topic question arises here for me: why not

- do away with the `x` and `y` arguments in the `__init__` or
- assign the values now that we have them.

Obviously this is not related to your change, I only noticed it just now.

Reply:
@Ditti4 right, the x, y arguments are useless and can be removed - why not send another PR?